src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -243,12 +238,12 @@ private string FindServerScriptFile()
 
         private static string GetQuotedScriptPath(string scriptPath)
         {
-            const string Quote = "\"";
+            const string quote = "\"";
```

Comment:
Cosmetics, my resharper was complaining that it's not the naming convention for local consts